### ISSUE TITLE
NVDA+Firefox: NVDA doesn't read second span in the <button> element after dialog is collapsed

### ISSUE THREAD

**Issue by Leuvarden** [nvaccess/nvda#11755]
### Steps to reproduce:

1. Open the code snippet https://codepen.io/leuvarden/pen/vYKNYMe?editors=1010 in Firefox 81.0.1 with NVDA on
2. Tab to "Open Modal", press Enter
3. Close the dialog by pressing Escape or clicking the "Close" button

### Actual behavior:

NVDA ignores second text of the visible <span> in the menu
![image](https://user-images.githubusercontent.com/16884735/95853489-30b22400-0d5e-11eb-8b52-0c95d5fa7105.png)

### Expected behavior:

NVDA reads all text content of the button

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA 2020.2

#### Windows version:
Microsot Windows 10 Enterprise
10.0.18363 Build 18363

#### Name and version of other software in use when reproducing the issue:
Firefox 81.0.1 (32-bit)

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If addons are disabled, is your problem still occuring?
Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by Adriani90** [comment]
This works as expected with NVDA 2024.1 Beta 13 and Firefox 123 as well as Chrome and Edge 122. Closing as works for me. if you are still having this issue please comment and we can reopen.
